### ISSUE TITLE
Can I use background.svg?

### ISSUE THREAD

**Issue by franciscop** [tcfraser/tcfraser.github.io#1]
I stumbled upon background.svg by doing a low polygon image search and I really love it. I am thinking of using it as a background image for the demo and maybe for the main page (if I redesign it) of [Picnic CSS](http://picnicss.com/), a small open source CSS library I made.

What license do you have for this image? I couldn't find any `LICENSE` in the repository.

**Comment by tcfraser** [comment]
Glad to see you like it!

I made it a couple weeks ago. Yeah of course you can use it for anything you'd like! If you do decide to redesign your main page, send me an email or message me so I can take a look.

I'll add a `LICENSE` this weekend some time but don't worry you can use it in anyway you want.

If you need any other versions of it let me know.

PS
If it makes sense for you website, and you feel like attributing me that'd be cool, but I don't require it.

**Comment by franciscop** [comment]
Nice! I will surely attribute it (however I'm still not sure how), I am using some open source tools from other people and I think it's the least I can do. I guess I should attribute your website, [http://tcfraser.com/](http://tcfraser.com/), right? I'll let you know when I change it in few days/weeks, thank you!

PS, I'm also doing a low-poly generator in Javascript as a side project as I really like this kind of images. Did you post anywhere how you created this one? While I have seen [some other nice ones](https://d13yacurqjgara.cloudfront.net/users/2973/screenshots/1587932/ble_n_01.jpg), this is the one that I've found that suits better as a background instead of a stand alone image.

**Comment by tcfraser** [comment]
Given that you're interested, I'll write up a little post about it and send you a link. Essentially I used a program called [DMesh](http://dmesh.thedofl.com/) on an actually image of a galaxy. Then I spent a few hours in Inkscape adjusting the triangles and their density to make it exactly the way I wanted. Then added a glow effect, adjusted the palette, and increased saturation. I have written simple low-poly generators in the past and I find that it never turns out the way I want. Usually I have to edit everything by hand afterward. 

I'm sure you'll figure out something for attributions in the future. Maybe at the end of an *About* page or something.

Currently this image is set to many of my profile pictures, desktop background etc.

Also, how did you find the image anyway? I am certain my Github Page and website gets essentially zero traffic.

**Comment by franciscop** [comment]
I found it by doing this google image search: `low poly filetype:svg`

Your image was the 8th one displaying for me. Also, your image was the only one in the search that could qualify as a background and luckily you also had a github and wanted to share it!

I made a few tweaks since the center colors were a bit too bright for the white text, and this is the final output for the library (still not published):

![new picnic web](https://cloud.githubusercontent.com/assets/2801252/7784466/d5581f9c-01a1-11e5-9e21-c58fa23c6596.png)

Thanks to your image, the total page weight is super-light under 100kb. And it certainly looks stunning (:
